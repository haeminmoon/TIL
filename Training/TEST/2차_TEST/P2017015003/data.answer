Case #1
2
Case #2
2
Case #3
4
Case #4
2
Case #5
2
Case #6
13
Case #7
5
Case #8
3
Case #9
4
Case #10
2
Case #11
2
Case #12
5
Case #13
2
Case #14
2
Case #15
4
Case #16
5
Case #17
3
Case #18
2
Case #19
2
Case #20
2
Case #21
3
Case #22
9
Case #23
2
Case #24
3
Case #25
45
Case #26
30
Case #27
3
Case #28
5
Case #29
10
Case #30
2
Case #31
3
Case #32
12
Case #33
10
Case #34
2
Case #35
112
Case #36
17
Case #37
2
Case #38
10
Case #39
5
Case #40
2
Case #41
1
Case #42
26
Case #43
2
Case #44
15
Case #45
4
Case #46
5
Case #47
3
Case #48
4
Case #49
3
Case #50
3
Case #51
2
Case #52
165
Case #53
35
Case #54
3
Case #55
2
Case #56
42
Case #57
2
Case #58
2
Case #59
2
Case #60
2
Case #61
2
Case #62
9
Case #63
2
Case #64
2
Case #65
2
Case #66
2
Case #67
60
Case #68
3
Case #69
5
Case #70
3
Case #71
11
Case #72
2
Case #73
10
Case #74
70
Case #75
7
Case #76
4
Case #77
2
Case #78
2
Case #79
4
Case #80
2
Case #81
342
Case #82
4
Case #83
3
Case #84
4
Case #85
2
Case #86
2
Case #87
10
Case #88
12
Case #89
2
Case #90
4
Case #91
3
Case #92
2
Case #93
3
Case #94
8
Case #95
36
Case #96
2
Case #97
2
Case #98
2
Case #99
2
Case #100
3
