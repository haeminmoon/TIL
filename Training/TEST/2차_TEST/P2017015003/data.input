100
15 3 28
15 12 169
5 2 3
18 19 305
19 13 222
14 7 8
13 19 58
8 7 22
8 2 5
4 13 27
8 10 61
14 8 25
6 12 49
18 15 241
6 2 3
15 11 34
7 19 58
9 2 15
3 15 31
9 12 61
18 4 29
20 3 7
6 18 91
19 4 29
9 5 1
6 5 1
4 2 3
6 3 4
13 3 4
18 7 71
3 5 6
1 12 1
19 10 21
20 5 81
14 8 1
1 17 1
12 13 131
10 20 21
10 4 9
20 2 23
5 1 5
2 13 1
19 9 127
1 15 1
13 4 13
5 15 16
20 7 50
16 13 66
16 1 7
11 3 13
10 12 109
15 11 1
7 5 1
9 16 65
15 17 188
3 14 1
6 17 52
12 4 33
6 6 19
12 13 79
2 17 18
9 10 11
8 15 106
16 2 21
14 10 91
14 1 13
3 20 1
15 14 85
14 6 19
13 3 13
13 5 6
3 2 3
11 6 7
7 10 1
20 16 49
14 7 29
16 12 181
9 5 36
7 9 19
10 10 71
19 18 1
16 17 69
11 12 61
4 4 5
11 1 8
20 5 86
14 2 3
2 6 1
18 20 201
10 12 37
5 4 9
19 7 92
16 18 109
8 13 14
4 9 1
7 12 73
20 1 16
16 8 105
10 20 121
13 4 25
