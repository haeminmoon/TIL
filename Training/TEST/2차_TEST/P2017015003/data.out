Case #1
1
Case #2
1
Case #3
1
Case #4
1
Case #5
1
Case #6
11
Case #7
10
Case #8
12
Case #9
1
Case #10
5
Case #11
20
Case #12
14
Case #13
15
Case #14
16
Case #15
1
Case #16
5
Case #17
23
Case #18
23
Case #19
25
Case #20
4
Case #21
6
Case #22
21
Case #23
23
Case #24
26
Case #25
21
Case #26
6
Case #27
10
Case #28
18
Case #29
22
Case #30
34
Case #31
29
Case #32
18
Case #33
22
Case #34
30
Case #35
46
Case #36
19
Case #37
25
Case #38
38
Case #39
1
Case #40
44
Case #41
28
Case #42
36
Case #43
26
Case #44
21
Case #45
21
Case #46
20
Case #47
39
Case #48
16
Case #49
1
Case #50
34
Case #51
18
Case #52
42
Case #53
29
Case #54
48
Case #55
14
Case #56
44
Case #57
36
Case #58
41
Case #59
25
Case #60
31
Case #61
33
Case #62
29
Case #63
30
Case #64
41
Case #65
16
Case #66
43
Case #67
39
Case #68
43
Case #69
15
Case #70
30
Case #71
20
Case #72
1
Case #73
10
Case #74
37
Case #75
34
Case #76
46
Case #77
16
Case #78
35
Case #79
28
Case #80
38
Case #81
37
Case #82
52
Case #83
27
Case #84
33
Case #85
27
Case #86
51
Case #87
11
Case #88
24
Case #89
48
Case #90
28
Case #91
17
Case #92
46
Case #93
44
Case #94
50
Case #95
25
Case #96
27
Case #97
31
Case #98
34
Case #99
37
Case #100
19
